bilge-impl/src/bitsize_internal.rs

```diff
@@ -237,3 +279,48 @@ fn generate_common(ir: ItemIr, arb_int: &TokenStream) -> TokenStream {
         }
     }
 }
+
+/// generate std::fmt impls - currently only std::fmt::Binary
+fn generate_enum_fmt_impls(variants: Iter<Variant>, enum_name: &Ident, arb_int: &TokenStream, bitsize: BitSize) -> TokenStream {
+    let to_int_match_arms = generate_to_int_match_arms(variants, enum_name, bitsize, arb_int);
+    
+    let body = if to_int_match_arms.is_empty() {
```

Comment:
I'm guessing this is can't happen anymore when #45 lands?

Reply:
it can - the function would just need to take an `Option<Fallback>`. whether it *should* is another question (2 of the 3 paths that would use this function don't care about fallback variants). personally I think it's okay.